Classes/Domain/Model/Library.php

```diff
@@ -147,6 +148,12 @@ class Library
      */
     protected $libraryDependencies;
 
+    /**
+     * @var Collection<LibraryDependency>
+     * @ORM\OneToMany(mappedBy="requiredLibrary", cascade={"persist", "remove"})
+     */
+    protected $librariesUsingThisLibrary;
```

Comment:
I realized that we were displaying the wrong thing (namely $libraryDependencies) in the "libraries using it" column in library/index.html. Instead I introduced a new reference here mapping to the correct column in the LibraryDependecy table. Please review if my thought process is correct here.